### DISCUSSION TITLE
How to pass translation keys in #[Title()] attributes

### DISCUSSION THREAD

**Discussion by valpuia604** [livewire/livewire#6358] [1. Help]
I want to change the title using selected locale translations, if I do like below then I can get
```php
public function render()
{
    return view('livewire.counter')
        ->title(trans('abc'));
}
```

But in attributes, I think it only accept string so both below code got error saying "Constant expression contains invalid operations"
```php
#[Title(trans('abc'))]
public function render()
{
    return view('livewire.counter');
}
```
```
public $title;

public function mount()
{
    $this->title = trans('abc');
}

#[Title($this->title)]
public function render()
{
    return view('livewire.counter');
}
```
Can I pass translations key in title attribute?

**Comment by joshhanley** [comment]
@valpuia604 if you need translations, then I would use the `->title()` method in render.
